libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -529,6 +529,14 @@ public void onUndoEnabled() {
                                               getResources().getInteger(android.R.integer.config_mediumAnimTime));
     }
 
+    @Override
+    public void onUndo() {
+    }
```

Comment:
Hey @daniloercoli, nice work here.
Maybe we can put some comment on those methods which explains why they don't have an implementation. WDYT ?

Reply:
Yes, good suggestion. Addressed in ecc84afe6978e94499c88cbbcd595822d5d8ed99

Reply:
Thanks!